index.bs

```diff
@@ -3844,19 +3850,36 @@ The [=remote end steps=] with |command parameters| are:
    1. Otherwise, set the |context|'s [=layout viewport=] to the
       implementation-defined default.
 
+1. Run the [[cssom-view-1#resizing-viewports]] steps.
+
 1. If |command parameters| [=map/contains=] the <code>devicePixelRatio</code> field:
 
    1. Let |device pixel ratio| be the |command
       parameters|["<code>devicePixelRatio</code>"].
 
-   1. If |device pixel ratio| is not null, change the size of the [=CSS Pixel=]
-      in |context|'s [=layout viewport=] such that it corresponds to |device
-      pixel ratio| in device pixels.
+   1. For the |context| and all descendant traversables:
```

Comment:
> all descendant traversables

this means iframes, right?

Reply:
Ah, that what you meant, here yes, it means all the children iframes.
(I thought yesterday you were talking about changing dpr only in an isolated iframe).

Reply:
@jgraham, what do you think?

Reply:
> The HTML spec says:
> 
> > Currently, all [traversable navigables](https://html.spec.whatwg.org/multipage/document-sequences.html#traversable-navigable) are [top-level traversables](https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversable). Future proposals envision introducing non-top-level traversables.
> 
> so if I read it right it cannot be iframes. Perhaps, we should link to https://html.spec.whatwg.org/multipage/document-sequences.html#descendant-navigables here?

I've checked again, and I think you're right, so I've replaced it to `descendant navigables`.